src/helpers/ValidMove.test.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import {validMove} from './ValidMove';
+
+test('Are there a function to validMove', () => {
+   expect(validMove()).toBeDefined;
+});
+
+test('Validate move with validMove function', () => {
+   expect(validMove('Qe4', 'rnbqkbnr/pppppppp/8/8/8/8/PPPPPPPP/RNBQKBNR')).toBeTruthy();

```

Comment:
In this case it will be Falsy not Truthy. Change that and add test cases for every piece, for every chess rule you know. Test both valid and invalid moves.